internal/services/cosmos/cosmosdb_restorable_database_account_data_source.go

```diff
@@ -0,0 +1,77 @@
+package cosmos
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/location"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/cosmos/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/cosmos/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+func dataSourceCosmosDbRestorableDatabaseAccount() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Read: dataSourceCosmosDbRestorableDatabaseAccountRead,
+
+		Timeouts: &pluginsdk.ResourceTimeout{
+			Read: pluginsdk.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*pluginsdk.Schema{
+			"name": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ValidateFunc: validate.CosmosAccountName,
+			},
+
+			"location": location.SchemaWithoutForceNew(),
+
+			"restorable_db_account_ids": {
```

Comment:
why limit ourselves to account?
```suggestion
			"accounts": {
			id
			deletion time
			creation time
			location
			restorable locations
			
```

can we grab all the relevant data a user might want?

Reply:
Updated